src/rle.js

```diff
@@ -18,8 +18,27 @@
  */
 
 function rle(source) {
-  // write code here
+  let counter = 0;
+  let result = '';
 
+  for (let i = 0; i < source.length; i++) {
+    let current = source[i];
+    let next = source[i + 1];
```

Comment:
will this fail if we pass source of single character?

Reply:
@roman-mate thank you for pointing it up. Will add the test-case with a single character.